### DISCUSSION TITLE
Expander Card

### DISCUSSION THREAD

**Discussion by milandzuris** [home-assistant/frontend#18931] [Dashboard]
Make this Card please thx, or someone fix it and put it on HACS.

https://github.com/Alia5/lovelace-expander-card/

https://github.com/home-assistant/frontend/assets/42905545/360f9582-60e1-4395-8e2e-715d5c8998ca

**Comment by ildar170975** [comment]
Is it a custom card?
If yes - you should ask in a corr. repo.

**Comment by IIIdefconIII** [comment]
can we please get this card also working with card mod?

![image](https://github.com/user-attachments/assets/8903a964-a177-4a71-80f8-fa3f980363b5)
